apps/frontend/src/modules/users/users-api.ts

```diff
@@ -20,6 +20,19 @@ class UserApi extends BaseHTTPApi {
 		super({ baseUrl, http, path: APIPath.USERS, storage });
 	}
 
+	public async getById(id: number): Promise<UserAuthResponseDto> {
+		const response = await this.load(
+			this.getFullEndpoint(`${UsersApiPath.ROOT}${id}`, {}),
```

Comment:
```suggestion
			this.getFullEndpoint(UsersApiPath.$ID, { id: String(id)}),
```
Isn't it better this way? As far as I understand the second argument is exactly for parameters